lib/common/client/base.rb

```diff
@@ -31,25 +31,26 @@ def perform(method, path, params, headers = nil)
 
       def request(method, path, params = {}, headers = {})
         raise_not_authenticated if headers.keys.include?('Token') && headers['Token'].nil?
+        binding.pry
```

Comment:
ahh, yea it's big time WIP -- just thought I'd push up changes early in case it might help your open PR.